fastlane/swift/LaneFileProtocol.swift

```diff
@@ -107,13 +111,16 @@ public class LaneFile: NSObject, LaneFileProtocol {
         let lowerCasedLaneRequested = named.lowercased()
 
         guard let laneMethod = currentLanes[lowerCasedLaneRequested] else {
-            let laneNames = self.laneFunctionNames.map { laneFuctionName in
-                if laneFuctionName.hasSuffix("lanewithoptions:") {
-                    return trimLaneWithOptionsFromName(laneName: laneFuctionName)
-                } else {
-                    return trimLaneFromName(laneName: laneFuctionName)
-                }
-            }.joined(separator: ", ")
+			let laneNames = self.laneFunctionNames.compactMap { functionName in
+				let laneFunctionName = functionName.lowercased()
+				if laneFunctionName.hasSuffix("lane") {
+					return trimLaneFromName(laneName: laneFunctionName)
+				} else if laneFunctionName.hasSuffix("lanewithoptions:") {
```

Comment:
Should we remove the `:` from here since the lane one doesn't have a `:`?

Reply:
A lane with parameters for example ```func thirdLane(withOptions options:[String: String]?) { ... }``` is going to have the following signature: ```thirdLaneWithOptions:```, that's why removing 4 characters, it was left ```thirdLaneWithOpti```

Reply:
That's why I have presented the examples above, and I have done tests with different method signatures.

Reply:
More details about the tests I have done here:
https://medium.com/@doruvil/fatlane-continous-integration-swift-version-b8851f1f38b6